server/db/hooks.js

```diff
@@ -8,6 +8,11 @@ const {
     Review,
 } = require('./relationships')
 
-//export models
+Category.getProducts = function (name) {
+    return Category.findOne({
+        where: name,
+        include: { model: Product, through: { attributes: [] } },
+    })
+}
```

Comment:
Isnt this already a default method in a hasmany association? I think you can just do catinstance.getProducts() and it returns all the products of that category.

Reply:
Yes it is. I changed the code a round to use that instead and just moved it to the Category Model.js file. Let me know if you think that works better